src/services/JobRoleService.ts

```diff
@@ -18,14 +19,14 @@ export const getJobs = async (): Promise<JobRole[]> => {
     }
 }
 
-export const getJobDetailsById = async (id: string): Promise<JobRoleDetailedResponse[]> => {
+export const getJobDetailsById = async (id: string, token: string): Promise<JobRoleDetailedResponse[]> => {
     try {
-        const response: AxiosResponse = await axios.get(URL+id)
+        const response: AxiosResponse = await axios.get(URL+id, getHeader(token))
+
         return response.data;
     } catch (e) {
```

Comment:
This looks like it is undoing the changes that were made for Job Role Description ??  

Maybe this just needs the latest main branch merged into this feature branch ??

Reply:
Fixed

Reply:
perfect, thanks...